skimage/feature/template.py

```diff
@@ -43,10 +43,10 @@ def match_template(image, template, pad_input=False, mode='constant',
 
     Parameters
     ----------
-    image : (M, N[, D]) array
+    image : ([P, ]M, N) array
         2-D or 3-D input image.
-    template : (m, n[, d]) array
-        Template to locate. It must be `(m <= M, n <= N[, d <= D])`.
+    template : ([p, ]m, n) array
+        Template to locate. It must be `(m <= M, n <= N[, p <= P])`.
```

Comment:
The ordering of the inequalities should match the ordering of the dimensions in the shape.

Reply:
@jni it looks like it does, doesn't it?

Reply:
Maybe my comment relates to an earlier version? 🤷 anyway, all good! 👍